machine/machine_test.go

```diff
@@ -193,8 +194,8 @@ func TestReadWriteAt(t *testing.T) {
 		t.Fatalf("ReedAt(b, %#x): (%d,%v) != (%d,nil)", off, n, err, len(b))
 	}
 
-	if !bytes.Equal(b[:], []byte(machine.Poison)[:4]) {
-		t.Fatalf("ReadAt(b, %#x): %#x != %#x", off, b, machine.Poison)
+	if !bytes.Equal(b[:], []byte(memory.Poison)[:4]) {
+		t.Fatalf("ReadAt(b, %#x): %#x != %#x", off, b, memory.Poison)
```

Comment:
Since poisoning is now in the scope of the memory package, I think it is appropriate to test it in memory_test.go.